portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,42 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/data1") //"data" kept loading a ghost file I had deleted so I came up with this stupid workaround
```

Comment:
Can you elaborate more on what the ghost file is? Usually we don't put a comment like this in the file :)

Reply:
What happened was I was playing around with static html elements, and thought that I would actually need a "/data" page to mix static element with fetched elements. The page was blank and had nothing but my portfolio background. Realizing I misunderstood how to do what I was trying to do, I deleted the actual page. However, every time I accessed /data via the servlet, instead of printing "Hello Uri" as it was supposed to, it brought up just the background from the /data page, even though it had been deleted. So, I came up with the workaround of just doing /data1.